api/api/clubs/serializers.py

```diff
@@ -6,4 +6,7 @@
 class ClubSerializer(serializers.ModelSerializer):
     class Meta:
         model = Club
-        fields = ['name', 'category', 'logo']
+        fields = ['name', 'category', 'logo', 'description', 'email']
```

Comment:
update URL uses `<int:pk>` for `updation` . We need to send `Club id` to the Client